README.md

```diff
@@ -347,6 +367,22 @@ The **plasmidfinder.tsv** output file generated by `staramr` contains the follow
 | SRR1952908 | ColpVC    | 98.96     | 100      | 193/193                 | contig00038 | 1618  | 1426  | JX133088  |
 | SRR1952908 | IncFIB(S) | 98.91     | 100      | 643/643                 | contig00024 | 10302 | 9660  | FN432031  |
 
+## mlst.tsv
```

Comment:
Could you add an entry for **mlst.tsv** to the table of contents near the beginning of this file?

Reply:
Could you clarify a bit more in what you mean by this? I've showned an `mlst.tv` table under the examples unless its not on this commit

Reply:
I mean adding an entry for `mlst.tsv` here https://github.com/phac-nml/staramr/tree/update_docs_0.6.0#table-of-contents

![image](https://user-images.githubusercontent.com/200517/64626313-43b43100-d3b3-11e9-97ed-445671e96cb0.png)

Similar to how to the other **Output** files have a specific entry in the table of contents.

Reply:
I see, added this in commit 3269fe1